src/tests/ftest/util/results_utils.py

```diff
@@ -333,7 +348,7 @@ def create_xml(job, results):
     # pylint: disable=import-outside-toplevel
     from avocado.plugins.xunit import XUnitResult
     result_xml = XUnitResult()
-    result_xml.render(results, job)
+    result_xml.render(sanitize_results(results), job)
```

Comment:
`sanitize_results(results)` doesn't return anything

Reply:
I think I beat you to it :)